client/src/featureform/sqlite_metadata.py

```diff
@@ -194,13 +193,19 @@ def getVariantResource(self, type, column, resource):
         variant_table_query = "SELECT * FROM " + type + " WHERE " + column + "='" + resource + "';"
         variant_data = self.__conn.execute(variant_table_query)
         self.__conn.commit()
+        variant_data_list = variant_data.fetchall()
+        if len(variant_data_list) == 0:
+          raise ValueError("{} with {}: {} not found".format(type, column, resource))
         return variant_data.fetchall()
 
     def getNameVariant(self, type, column1, resource1, column2, resource2):
         variant_table_query = "SELECT * FROM " + type + " WHERE " + column1 + "='" + resource1 + "' AND " + column2 + "='" + resource2 + "';"
         variant_data = self.__conn.execute(variant_table_query)
         self.__conn.commit()
-        return variant_data.fetchall()
+        variant_data_list = variant_data.fetchall()
+        if len(variant_data_list) == 0:
+          raise ValueError("{} with {}: {} and {}: {} not found".format(type, column1, resource1, column2, resource2))
```

Comment:
For future reference, leveraging f-strings makes it a lot easier to understand.
Here is how this one would look:
```
f"{type} with {column1}: {resource1} and {column2}: {resource2} not found"
```